### PR TITLE
5.2.0

### PR DESCRIPTION
**Pull request by trentm**
And some packaging/release tweaks: "files" to limit what files are
included in the published package; 'make cutarelease' convenience.

### CODE REVIEW

**Review by astorm — APPROVED** [review]
Didn't dive deep into the specifics but a little more process around release here sounds like a positive change to me.  Approving, pending the question that github repo being in the path.

File: Makefile
Comment (trentm) [reply Makefile:2]: Yes, it is still relevant. I could add the (single file) `json` utility locally in the repo and use that directly, if you think that would be worthwhile. The only downside here is that a maintainer that doesn't have `json` installed will not be able to do a release using the `make cutarelease` convenience.

Comment (trentm) [reply Makefile:2]: Dunno if a MAINTAINERS.md or some other named file or a section in the README.md for maintainers on how to do a release would be helpful/needed as well?

Comment (trentm) [reply Makefile:2]: Perhaps a CONTRIBUTING.md (a la apm-agent-nodejs.git) with a section on how to do a release?

Comment (astorm) [reply Makefile:2]: All good -- I just didn't know if you wanted your personal repo hanging out in a comment in a work repo. 👍

**Comment by trentm** [comment]
require-in-the-middle@5.2.0 published

**Pull request merged**
